python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -301,10 +301,10 @@ class LogisticRegression(Base, ClassifierMixin):
             )
 
         if self.fit_intercept:
-            self.coef_ = self.qn.coef_[0:-1]
-            self.intercept_ = self.qn.coef_[-1]
+            self.coef_ = self.solver_model.coef_[0:-1]
+            self.intercept_ = self.solver_model.coef_[-1]
```

Comment:
This happens in qn here https://github.com/rapidsai/cuml/blob/133a57fd0ca97b00855218fb311f523800ec579a/python/cuml/solvers/qn.pyx#L524

Though it seems that the intercept is not really removed from the coefs there, which seems like a bug we could fix here, what do you think?

Reply:
I meant that in qn, I don't see the coef_ being assigned to the coef[0:-1] (only intercept to coef[-1])while here they are. So that seems like an unintended discrepancy, and if we do that in qn, we can skip doing it here and rely on the coefs/intercept of the solver_model

Reply:
So, here's the issue - when I try to include 
```
def __getattr__(self, attr):
        if attr == 'intercept_':
            if self.fit_intercept:
                return self.coef_[-1]
            else:
                return CumlArray.zeros(shape=1)
        elif attr == 'coef_':
            if self.fit_intercept:
                return self.coef_[0:-1]
            else:
                return self.coef_
        else:
            raise AttributeError(attr)
```

Because of my logic in `base.pyx` to handle `solver_model` it always returns `self.coef_` regardless of `fit_intercept` value if I move everything to `qn` (this is incorrect) for Logistic Regression it seems like it would be better to at least store the `coef_` in class instead of relying on `solver_model`